resource-managers/kubernetes/core/src/main/scala/org/apache/spark/scheduler/cluster/k8s/ExecutorPodsAllocator.scala

```diff
@@ -37,6 +37,8 @@ private[spark] class ExecutorPodsAllocator(
     snapshotsStore: ExecutorPodsSnapshotsStore,
     clock: Clock) extends Logging {
 
+  private val EXIT_MAX_EXECUTOR_FAILURES = 10
```

Comment:
Sorry, but why do you choose `10` as an exit code?
This conflicts with the existing exit value `EXIT_UNCAUGHT_EXCEPTION`.
I guess you want to have a unique value.

Reply:
Maybe it's better to reuse YARN's EXIT_MAX_EXECUTOR_FAILURES=11.